src/hammer-vlsi/hammer_vlsi/hammer_vlsi_impl.py

```diff
@@ -592,17 +602,26 @@ def env_vars(self) -> Dict[str, str]:
             "MGLS_LICENSE_FILE": self.get_setting("synopsys.MGLS_LICENSE_FILE")
         }
 
+    def version_number(self, version: str) -> int:
+        """
+        Assumes versions look like NAME-YYYY.MM-SPMINOR.
+        Assumes less than 100 minor versions.
+        """
+        date = "-".join(version.split("-")[1:])  # type: str
+        year = int(date.split(".")[0])  # type: int
+        month = int(date.split(".")[1][:2])  # type: int
+        minor_version = 0  # type: int
+        if "-" in date:
+            minor_version = int(date.split("-")[1][2:])
+        return (year * 100 + month) * 100 + minor_version
+
     def get_synopsys_rm_tarball(self, product: str, settings_key: str = "") -> str:
         """Locate reference methodology tarball.
 
         :param product: Either "DC" or "ICC"
         :param settings_key: Key to retrieve the version for the product. Leave blank for DC and ICC.
         """
-        key = settings_key # type: str
-        if product == "DC":
-            key = "synthesis.dc.dc_version"
-        elif product == "ICC":
-            key = "par.icc.icc_version"
+        key = self.tool_config_prefix() + "." + "version" # type: str
```

Comment:
Is `product` obsolete? If so we should remove the parameter.

Reply:
No we still use product to come up with the tarball name a few lines below.